app/Policies/MessagePolicy.php

```diff
@@ -46,4 +48,25 @@ public function forceDelete(User $user, Message $message): bool
     {
         return false;
     }
+
+    public function sendToRecipient(User $user, User $targetUser): bool
+    {
+        $isUserDeveloper = $user->hasAnyRole([
+            Role::DEVELOPER,
+            Role::DEVELOPER_JUNIOR,
+            Role::DEVELOPER_STAFF,
```

Comment:
Should this also include MODERATOR?

Reply:
It should, as well as a few others probably. Updated in latest to:
```php
$canUserAlwaysSend = $user->hasAnyRole([
    Role::ADMINISTRATOR,
    Role::DEVELOPER_JUNIOR,
    Role::DEVELOPER_STAFF,
    Role::DEVELOPER,
    Role::EVENT_MANAGER,
    Role::FORUM_MANAGER,
    Role::MODERATOR,
    Role::POINT_OF_CONTACT,
])
    || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
```